apps/vaults/components/details/actions/QuickActionsTo.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {useActionFlow} from '@vaults/contexts/useActionFlow';
+import {useSolver} from '@vaults/contexts/useSolver';
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
+import {formatCounterValue} from '@yearn-finance/web-lib/utils/format.value';
+import {Dropdown} from '@common/components/TokenDropdown';
+import {useTokenPrice} from '@common/hooks/useTokenPrice';
+import {formatPercent} from '@common/utils';
+
+import type {ReactElement} from 'react';
+import type {TDropdownOption} from '@common/types/types';
+
+function	VaultDetailsQuickActionsTo(): ReactElement {
+	const {
+		currentVault,
+		possibleOptionsTo, selectedOptionTo, onUpdateSelectedOptionTo,
+		isDepositing
+	} = useActionFlow();
```

Comment:
nit: Looks confusing otherwise
```suggestion
	const {currentVault, possibleOptionsTo, selectedOptionTo, onUpdateSelectedOptionTo, isDepositing} = useActionFlow();
```